.github/workflows/build.yaml

```diff
@@ -15,43 +15,41 @@ jobs:
             - name: Validate composer.json
               run: composer validate --strict --no-check-lock
     cs-fixer:
-        runs-on: ubuntu-20.04
+        runs-on: ubuntu-22.04
         name: PHP-CS-Fixer
         steps:
             - name: Checkout
               uses: actions/checkout@v3
             - name: Fix CS
               uses: docker://oskarstark/php-cs-fixer-ga
     tests:
-        runs-on: ubuntu-20.04
+        runs-on: ubuntu-22.04
         strategy:
             fail-fast: false
             matrix:
                 include:
-                    - description: 'No Symfony specified'
-                      php: '7.4'
                     - description: 'No Symfony specified'
                       php: '8.0'
                     - description: 'No Symfony specified'
                       php: '8.1'
                     - description: 'No Symfony specified'
                       php: '8.2'
                     - description: 'Lowest deps'
-                      php: '7.4'
+                      php: '8.0'
                       composer_option: '--prefer-lowest'
                       env:
                           SYMFONY_DEPRECATIONS_HELPER: max[self]=0
-                    - description: 'Symfony 4.4'
-                      php: '7.4'
-                      symfony: 4.4.*
                     - description: 'Symfony 5.4'
-                      php: '7.4'
-                      symfony: 5.4.*
-                    - description: 'Symfony 6.0'
                       php: '8.0'
-                      symfony: 6.0.*
-                    - description: 'Dev deps'
+                      symfony: 5.4.*
+                    - description: 'Symfony 5.4'
```

Comment:
We don't need multiple jobs for Symfony 5.4

Reply:
I changed it, can you please have a look if it now ok?